code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -359,28 +359,37 @@
 							TEMPERANCE_ATTRIBUTE = 80
 							)
 	var/vine_cooldown
-	var/vine_delay = 1 SECONDS
+	var/vine_delay = 3 SECONDS
```

Comment:
Either increase the cooldown time, or move it below the ability itself.

Reply:
I think i can just remove it since vine_delay isnt supposed to be modified since its a cooldown